pkg/manager/runtime_env/bll.go

```diff
@@ -0,0 +1,41 @@
+package runtime_env
```

Comment:
bll means Business Logic Layer? The file name may not very good.

Reply:
Yes. Do you have any suggestions for file naming?

Reply:
Sorry, I do not have a better one, can these code be merged into handler.go?